### ISSUE TITLE
Windows Python artifact build broken

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#14317]
Not linked to a specific change, suddenly all windows artifact build are broken. I suspect some python package got updated on pip, which broke the build?

https://sponge.corp.google.com/invocation?tab=Build+Log&id=410d9145-8fbb-4439-baad-ae01f460eb0e&searchFor=

**Comment by mehrdada** [comment]
On Kokoro? We had a problem with Jenkins that @matt-kwong fixed recently

**Comment by jtattermusch** [comment]
```
[1/1] Cythonizing src\python\grpcio\grpc/_cython/cygrpc.pyx

running build_ext

C:\Python35\lib\site-packages\Cython\Distutils\old_build_ext.py:30: UserWarning: Cython.Distutils.old_build_ext does not properly handle dependencies and is deprecated.

  "Cython.Distutils.old_build_ext does not properly handle dependencies "

Traceback (most recent call last):

  File "setup.py", line 311, in <module>

    cmdclass=COMMAND_CLASS,

  File "C:\Python35\lib\site-packages\setuptools\__init__.py", line 129, in setup

    return distutils.core.setup(**attrs)

  File "C:\Python35\lib\distutils\core.py", line 148, in setup

    dist.run_commands()

  File "C:\Python35\lib\distutils\dist.py", line 955, in run_commands

    self.run_command(cmd)

  File "C:\Python35\lib\distutils\dist.py", line 973, in run_command

    cmd_obj.ensure_finalized()

  File "C:\Python35\lib\distutils\cmd.py", line 107, in ensure_finalized

    self.finalize_options()

  File "C:\Python35\lib\site-packages\setuptools\command\build_ext.py", line 133, in finalize_options

    _build_ext.finalize_options(self)

  File "C:\Python35\lib\site-packages\Cython\Distutils\build_ext.py", line 20, in finalize_options

    super(build_ext, self).finalize_options()

TypeError: super(type, obj): obj must be an instance or subtype of type
```

**Comment by jtattermusch** [comment]
@mehrdada , yes, on Kokoro.

**Comment by jtattermusch** [comment]
It started failing about 1 day ago

**Comment by jtattermusch** [comment]
It looks like all the Python 64bit artifacts are broken, all the 32bit work.
https://sponge.corp.google.com/target?showTestCases=FAILURE&showTestCases=PASSED&showTestCases=&id=410d9145-8fbb-4439-baad-ae01f460eb0e&searchFor=&show=ALL&tab=Test+Cases&target=github%2Fgrpc&view=FLAT&sortMethodsBy=STATUS&showRuns=SHOW_ALL_RUNS

**Comment by jtattermusch** [comment]
https://pypi.python.org/pypi/setuptools/38.5.0  was released yesterday.

**Comment by mehrdada** [comment]
That matches the symptoms on Jenkins last week.  I wonder how @matt-kwong managed to fix it?

**Comment by matt-kwong** [comment]
I don't think this is similar to the issue I fixed last week. Last week's issue was a specific Jenkins VM failing builds (I never figured out why), so I disabled it and let other VMs handle the Windows artifact jobs. I think this looks more like a pip dependency being upgraded and breaking our builds, which has happened multiple times in the past. Pinning it might fix the issue.

**Comment by jtattermusch** [comment]
Fixed by #14320
